### PR TITLE
[one-cmds-test] Do not remove reused circle file

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise not do remove reused output circle file.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #9710

**Review by jinevening — APPROVED** [review]
LGTM

**Comment by seanshpark** [comment]
@jinevening , I've added two more files. PTAL again.

File: compiler/one-cmds/tests/onecc_013.test [hunk 1]
```diff
@@ -30,7 +30,8 @@ trap trap_err_onexit ERR
 configfile="onecc_013.cfg"
 outputfile="inception_v3.circle"
 
-rm -rf ${outputfile}
+# do not remove output file
+# rm -rf ${outputfile}
```
Comment (jinevening) [inline compiler/one-cmds/tests/onecc_013.test:34]: Can I ask why outputfile is not removed in `onecc_013.test` and `onecc_014.test`?

If outputfile exists before test is started, the test will be always passed.

Comment (seanshpark) [reply compiler/one-cmds/tests/onecc_013.test:34]: this test conflicts with the problem deleing reused `inception_v3.circle` file.
I'll try to rename the output file

File: compiler/one-cmds/tests/onecc_013.cfg [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [one-import-tf]
 input_path=inception_v3.pb
-output_path=inception_v3.circle
+output_path=inception_v3_test.circle
```
Comment (jinevening) [inline compiler/one-cmds/tests/onecc_013.cfg:3]: How about `inception_v3.onecc_013.circle`?

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**